src/data_model/utils.jl

```diff
@@ -400,6 +402,7 @@ function _build_loss_model!(
             "b_to" => diagm(0=>fill(b_to, nphases)),
             "angmin" => fill(-10.0, nphases),
             "angmax" => fill( 10.0, nphases),
+            "c_rating_a" => fill(Inf, nphases),
```

Comment:
We need to think about how to propagate bounds to transformer math objects, so that we can easily run bounds-requiring formulations without a weird workflow

Reply:
#351